e2e/playwright/tests/@version-history-pagination/test.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { test, expect, Page } from '@playwright/test';
+import { login, uploadLicense } from '../shared';
+import { execSync } from 'child_process';
+
+test('version history pagination', async ({ page }) => {
+  test.slow();
+  test.setTimeout(300000); // 300 seconds, 5 minutes
+  const testAppSlug = "version-history-pagination";
```

Comment:
can you move these to a constants file and remove the `test` prefix (to match the pattern i have for regression tests)? something like this:
```suggestion
  export const APP_SLUG = "version-history-pagination";
```